semtypes/src/main/java/io/ballerina/types/subtypedata/BddNode.java

```diff
@@ -20,60 +20,26 @@
 import io.ballerina.types.Atom;
 import io.ballerina.types.Bdd;
 
-import java.util.Objects;
-import java.util.concurrent.atomic.AtomicInteger;
-
 /**
- * Bdd node.
+ * Internal node of a BDD, which represents a disjunction of conjunctions of atoms.
  *
+ * @param atom the atom that this node represents
```

Comment:
Shall we format param doc lines? Pram description will start at the same column position for all params

Reply:
Fixed